stable/entropy/values.yaml

```diff
@@ -52,3 +52,47 @@ app:
     # NEW_RELIC_APP_NAME: entropy
 
   secretConfig: {}
+
+workers:
```

Comment:
Let's also make it possible to put multiple worker deployments for this?

Reply:
this will require changes in the template, should we create a new template for the workers?

Reply:
hmm you are correct, I think we can keep this as it is, user could have multiple helm release if they need.

but the `workers` name is plural. isn't it a bit misleading?

Reply:
oh right, will use `worker` instead